templates/zerver/help/view-zulip-version.md

```diff
@@ -0,0 +1,46 @@
+# View Zulip version
+
+## Zulip server and web app version
+
+If your organization self-hosts Zulip, the server administrator will need to
+[upgrade Zulip][upgrade-zulip] in order for you to experience the latest
+improvements. You can check which [version][changelog] of the Zulip server and
+web app your organization is using.
+
+Zulip Cloud organizations are always updated to the latest version of Zulip.
+
+[upgrade-zulip]:
+    https://zulip.readthedocs.io/en/latest/production/upgrade-or-modify.html
+[changelog]: https://zulip.readthedocs.io/en/latest/overview/changelog.html
+
+### View Zulip server and web app version
+
+{start_tabs}
+
+1. Click on the **gear** (<i class="fa fa-cog"></i>) icon in the upper
+   right corner of the web or desktop app.
+
+1. Select **About Zulip** from the menu.
+
+1. View the version number under **Zulip Server**.
```

Comment:
We may want to mention the fact that if the Zulip server is running a branch rather than an official release, we'll have a separate "merge base" property displayed.  It feels complicated to bring up, but it's also the most useful value to see in a bug report for someone running such a branch.

Reply:
I applied this further edit:

```
     If the Zulip server is running a custom branch, the **merge base**
-    will also be shown. It indicates where the custom branch was split off from
-    the official release.
+    will also be shown. It indicates which official version of Zulip
+    the custom branch is based on.
```